style.css

```diff
@@ -102,28 +104,26 @@ h3 {
     text-align: center;
     font-size: 36px;
     font-weight: bolder;
-    color: #1F2937;
+    color: #1f2937;
 }
 
 .boxes {
-    background-color: #ffffff;
+    background-color: #fff;
     display: flex;
     justify-content: center;
-    height: 360px;
     gap: 64px;
+    margin: 0 auto 124px;
+    flex-wrap: wrap;
 }
 
 .boxBlock {
-    width: 160px;
+    max-width: 170px;
 }
 
 .box {
-    border: 2px solid #3882F6;
+    aspect-ratio: 1 / 1;
```

Comment:
Rather than a fixed width and height, the box now uses an aspect ratio - this will be more flexible for when the container width changes i.e. on smaller devices.